README.md

```diff
@@ -14,30 +14,42 @@ If there is any information that would be helpful but is missing from the change
-| Windows Server          | 2019  | Jan, 2024       | Sept, 2023 | [x86_64](windows%202019/windows_2019.md)                                  | |
-| Talos                   | v1    | Rolling Release | N/A        | [x86_64](talos/talos_v1.md) [arm64](talos/talos_v1.md) | Available | |
-| Flatcar Linux           |stable | Rolling release | N/A        | Rolling release                       | Available                 | |
-| Flatcar Linux           |beta   | Rolling release | N/A        | Rolling release                       | Available                 | |
-| FreeBSD                 | 13.2  | Jan, 2026       | Jan, 2026  | [x86_64](freebsd/x86_64/13_2.md) [arm64](freebsd/arm64/13_2.md)   | |
-| FreeBSD                 | 14.0  | Jan, 2029       | Jan, 2029  | [x86_64](freebsd/x86_64/14_0.md) [arm64](freebsd/arm64/14_0.md)   | |
-| VyOS                    | 1.3   | ???             | ???        | TBD                                   | Coming soon               | |
+| OS                       | Version | Upstream EOL    | EM Estimated EOL |  Changelog                                            | Notes |
+| ---                      | ---     | ---             | ---        | ---                                                           | --- |
+| Ubuntu                   | 20.04   | Apr, 2030       | Jan, 2030  | [x86_64](ubuntu/x86_64/20_04.md) [arm64](ubuntu/aarch64/20_04.md) | |
+| Ubuntu                   | 22.04   | Apr, 2032       | Jan, 2032  | [x86_64](ubuntu/x86_64/22_04.md) [arm64](ubuntu/aarch64/22_04.md) | |
+| Ubuntu                   | 24.04   | Apr, 2034       | Jan, 2034  | [x86_64](ubuntu/x86_64/24_04.md) [arm64](ubuntu/aarch64/24_04.md) | |
+| Debian                   | 11      | Jun, 2026       | Mar, 2026  | [x86_64](debian/x86_64/11.md) [arm64](debian/aarch64/11.md)       | |
+| Debian                   | 12      | Jun, 2028       | May, 2028  | [x86_64](debian/x86_64/12.md) [arm64](debian/aarch64/12.md)       | |
+| Red Hat Enterprise Linux | 8       | May, 2024       | May 2029   | [x86_64](rhel/x86_64/8.md)                                        | |
+| Red Hat Enterprise Linux | 9       | May, 2027       | May 2032   | [x86_64](rhel/x86_64/9.md)                                        | |
+| AlmaLinux OS             | 8       | May, 2024       | Feb, 2024  | [x86_64](alma/x86_64/8.md) [arm64](alma/aarch64/8.md)             | |
+| AlmaLinux OS             | 9       | May, 2027       | Feb, 2027  | [x86_64](alma/x86_64/9.md) [arm64](alma/aarch64/9.md)             | |
+| Rocky Linux              | 8       | May, 2024       | Feb, 2024  | [x86_64](rocky/x86_64/8.md) [arm64](rocky/aarch64/8.md)           | |
+| Rocky Linux              | 9       | May, 2025       | Feb, 2025  | [x86_64](rocky/x86_64/9.md) [arm64](rocky/aarch64/9.md)           | |
+| Alpine                   | 3       | Rolling Release | N/A        | [x86_64](alpine/x86_64/3.md) [arm64](alpine/aarch64/3.md)         | |
```

Comment:
Alpine isn't actually a rolling release, we just happen to only support the latest version https://endoflife.date/alpine, not sure if appropriate to change that here though so going to approve